src/Microsoft.PowerShell.Commands.Utility/commands/utility/PSBreakpointCreationBase.cs

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.ObjectModel;
+using System.IO;
+using System.Management.Automation;
+using System.Management.Automation.Internal;
+
+namespace Microsoft.PowerShell.Commands
+{
+    /// <summary>
+    /// Base class for Set/New-PSBreakpoint.
+    /// </summary>
+    public class PSBreakpointCreationBase : PSCmdlet
+    {
+        #region parameters
+
+        /// <summary>
+        /// The action to take when hitting this breakpoint.
+        /// </summary>
+        [Parameter(ParameterSetName = "Command")]
+        [Parameter(ParameterSetName = "Line")]
+        [Parameter(ParameterSetName = "Variable")]
+        public ScriptBlock Action { get; set; } = null;
+
+        /// <summary>
+        /// The column to set the breakpoint on.
+        /// </summary>
+        [Parameter(Position = 2, ParameterSetName = "Line")]
+        [ValidateRange(1, int.MaxValue)]
+        public int Column { get; set; } = 0;
+
+        /// <summary>
+        /// The command(s) to set the breakpoint on.
+        /// </summary>
+        [Alias("C")]
+        [Parameter(ParameterSetName = "Command", Mandatory = true)]
+        [ValidateNotNull]
+        public string[] Command { get; set; } = null;
+
+        /// <summary>
+        /// The line to set the breakpoint on.
+        /// </summary>
+        [Parameter(Position = 1, ParameterSetName = "Line", Mandatory = true)]
+        [ValidateNotNull]
```

Comment:
Please remove - the parameter is mandatory.

Reply:
Fixed.